docs/releasing-workflow-instructions.md

```diff
@@ -0,0 +1,154 @@
+- ``itisfoundation/webserver:staging-github-latest``
+
+### Instructions to generate a staging release
+
+1. Generate Github release tag
+
+  ```bash
+  git clone https://github.com/ITISFoundation/osparc-simcore.git
+  cd osparc-simcore
+  make release-staging name=SPRINTNAME version=VERSION (git_sha=OPTIONAL)
+  ```
+
+2. Adjust the list of changes if needed
+3. Press the **Publish release** button
+4. The CI will be automatically triggered and will deploy the staging release
+
+## Release process
+
+A released version of simcore, that is marked as such on the master branch by leveraging Github release tagging mechanism. The CI is triggered and will pull the marked staging docker images (by using the given git SHA or the latest staging images), and tag them as release images.
+**NOTE:** A release version is ALWAYS preceded by a staging version. The CI will fail if it does not find the corresponding staging version.
+Each docker build marked as released are tagged as:
+
+- itisfoundation/[image_name]:release-[CINAME]-latest
+- itisfoundation/[image_name]:release-[CINAME]-v[RELEASE_VERSION]-[BUILD_DATE]--[BUILD_TIME].GIT_SHA
+
+with:
+
+- CINAME the name of the CI
+- RELEASE_VERSION is a version number following semantic versioning (e.g. 1.0.0, 1.2.0, 1.2.1, ...)
+- BUILD_DATE is the build date
+- BUILD_TIME is the build time
+- GIT_SHA is the git SHA corresponding to the code used for building
+
+see [docs/img/git-release-workflow.svg](docs/img/git-release-workflow.svg)
+
+### Release example
+
+The team decides to release to production the lastest staging version of ``DAJIA`` sprint. Next release version, following semantic versioning and previous releases, is `v5.6.0`. The images will be retaged by github actions CI as:
+
+- ``itisfoundation/webserver:release-github-v5.6.0-2020-09-02--19-30.560eq50f3babb6537580c0e03b85b9a8209bbf10``
+- ``itisfoundation/webserver:release-github-latest``
+
+### Instructions to generate a release
+
+1. Generate Github release tag
+
+  ```bash
+  git clone https://github.com/ITISFoundation/osparc-simcore.git
+  cd osparc-simcore
+  make release-prod version=MAJ.MIN.PATCH (git_sha=OPTIONAL)
+  ```
+
+2. Adjust the list of changes if needed
+3. Press the **Publish release** button
+4. The CI will be automatically triggered and will deploy the staging release
+
+## Hotfix process
+
+A hotfix is **ALWAYS made from an already released version**. A branch is created from the tagged release version and the fix is implemented. Once it is ready the CI will trigger and build the *hotfix-* labelled docker images. Then, the image shall be tagged leveraging Github release mechanism. The CI will trigger again and pull the docker images (based on git SHA or latest hotfix image) and push a new release version.
+
+Each docker build marked as released are tagged as:
+
+- itisfoundation/[image_name]:release-[CINAME]-latest
+- itisfoundation/[image_name]:release-[CINAME]-v[RELEASE_VERSION]-[BUILD_DATE]--[BUILD_TIME].[GIT_SHA]
```

Comment:
I see no hotfix in there image name here

Reply:
yes the release in the end is a release.
the CI creates a hotfix- candidate release that is pushed to dockerhub